tools/wt-mcp/README.md

```diff
@@ -0,0 +1,229 @@
+# WiredTiger MCP Server
+
+A Model Context Protocol (MCP) server that provides tools to interact with WiredTiger databases. This server exposes WiredTiger functionality through MCP tools, allowing easy analysis and interaction with WiredTiger files and databases.
+
+## Overview
+
+The WiredTiger MCP Server is designed to provide capabilities for analysing, inspecting, and interacting with WiredTiger databases through a standardised protocol. This enables integration with AI assistants, agents and other tools that support the Model Context Protocol.
+
+### What is MCP?
+
+The Model Context Protocol (MCP) is a standardised way for AI assistants to interact with external tools, resources, and environments. It enables AI models to:
+
+- Execute code in various environments
+- Access and manipulate files
+- Interact with databases and APIs
+- Retrieve relevant information from large codebases
+
+For more information, visit [modelcontextprotocol.io](https://modelcontextprotocol.io).
+
+## Installation
+
+### Prerequisites
+
+- Python 3.13
+- uv (a fast Python package installer and resolver from Astral - see [astral.sh/uv](https://astral.sh/uv) for installation instructions)
+- WiredTiger Python API
+- MCP Python SDK
+
+### Setup
+
+0. Install `uv` if you haven't already. `uv` is a fast Python package installer.
+   You can install it using pip:
+   ```bash
```

Comment:
[MD031/blanks-around-fences](https://github.com/DavidAnson/markdownlint/blob/v0.38.0/doc/md031.md), this can be easily fixed throughout your file. Would be great to enforce it - but for another day.